src/PortfolioPlanning/Common/Services/BacklogConfigurationDataService.ts

```diff
@@ -46,7 +52,8 @@ export class BacklogConfigurationDataService {
             effortFieldRefName: projectEfforFieldRefName,
 
             orderedWorkItemTypes: portfolioLevelsData.orderedWorkItemTypes,
-            backlogLevelNamesByWorkItemType: portfolioLevelsData.backlogLevelNamesByWorkItemType
+            backlogLevelNamesByWorkItemType: portfolioLevelsData.backlogLevelNamesByWorkItemType,
+            workItemTypeMappedStates: workItemTypeMappedStates
```

Comment:
Instead of an array of `WorkItemTypeStateInfo`, you could create a map whose key is the work item type (in lower case)...